src/components/MobileMenu.tsx

```diff
@@ -0,0 +1,118 @@
+import styled from '@emotion/styled';
+import Link from 'next/link';
+import React, { FC } from 'react';
+import { useLockBodyScroll, useToggle } from 'react-use';
+import { CTA } from '.';
+import { MenuIcon } from './icons';
+
+const textAnimation = `
+  opacity: 0;
+
+  .visible & {
+    opacity: 1;
+    transition: opacity 500ms ease 100ms;
+  }
+`;
+
+const Menu = styled.div`
+  background-color: ${({ theme }) => theme.colors.workThumbnailBg};
+  height: 100%;
+  left: 0;
+  position: fixed;
+  top: 0;
+  transform: scale(0, 0);
+  transform-origin: top right;
+  transition: transform 100ms ease-in-out;
+  width: 100%;
+  z-index: -1;
+
```

Comment:
```suggestion
opacity: 0;
```
I believe you need to add this as a baseline before applying transition below

Reply:
I think that it would depend on what we want to do with the transition. Currently the transition is just a scale up from 0 (the text is a fade in, which is separate). Adding this opacity line causes the menu to never show (unless we also want to transition the opacity up to 1).